lib/include/fdoprot.h

```diff
@@ -117,19 +105,13 @@
 #define FDO_TYPE_DEVICE_CRED 6
 #define FDO_TYPE_HMAC 7
 
-// Report
-#define FDO_MGR_AGENT_SND_REPORT 52
-#define FDO_MGR_AGENT_RCV_REPORT_ACK 53
-
 // For restful URL mapping
+// TODO: Not used, needs to review
```

Comment:
should this comment be removed? What is the TODO?

Reply:
Updated in commit 1c24ccaba0e54c28b18fdfaa30a87b1cad086e07